.jenkins

```diff
@@ -186,13 +186,32 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
-                        sh 'echo $PATH && rm -rf build && mkdir -p build && cd build && cmake -D CMAKE_BUILD_TYPE=Debug -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=pgc++ -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" -D ARBORX_ENABLE_MPI=ON -D MPIEXEC_PREFLAGS="--allow-run-as-root" .. && make -j8 VERBOSE=1 && ctest --no-compress-output -T Test'
+                        sh 'rm -rf build && mkdir -p build'
+                        dir('build') {
+                            sh '''
+                              cmake \
+                                  -D CMAKE_BUILD_TYPE=Debug \
+                                  -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
+                                  -D CMAKE_CXX_COMPILER=pgc++ \
+                                  -D CMAKE_CXX_FLAGS="-Minform=warn" \
```

Comment:
I am not sure what this flags does. The documentation is very confusing. Should we use `warn`:
```
warn
    instructs the compiler to display warning, severe and fatal error messages.
```
or `inform`:
```
inform
    instructs the compiler to display all error messages (inform, warn, severe and fatal).
```